CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Changed
 
 - The SDK now uses logr as the default logger to unify the logging output with the controller-runtime logs. Users can still use a logger of their own choice. See the [logging doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/logging.md) on how the SDK initializes and uses logr.
+- `operator-sdk print-deps` prints Golang packages and versions expected by the Operator SDK version being used by an operator. Supplying `--as-file` prints packages and versions in Gopkg.toml format.
```

Comment:
We usually append a hyperlinked PR number at the end of the changed description. So this should have ([#772](https://github.com/operator-framework/operator-sdk/pull/772)) at the end